question.html

```diff
@@ -86,6 +86,21 @@ <h1 size="10">Questions for {{roomId}}?</h1>&nbsp;
 	<!-- iterate questions -->
 	<span class="list-group-item" ng-repeat="todo in todos | filter:input | questionFilter:maxQuestion" ng-class="{completed: todo.completed, editing: todo == editedTodo}">
 		<!-- display questions -->
+		<!-- CSS style for questions -->
```

Comment:
Same here as in PR: Master #14
It should be possible to change the CSS layout of the class "list-group-item" in the CSS file. 
We should avoid placing large styling blocks in the html files.